kmath-geometry/src/commonMain/kotlin/scientifik/kmath/geometry/Euclidean2DSpace.kt

```diff
@@ -27,32 +28,29 @@ interface Vector2D : Point<Double>, Vector, SpaceElement<Vector2D, Vector2D, Euc
     override fun Vector2D.wrap(): Vector2D = this
 }
 
-val Vector2D.r: Double get() = Euclidean2DSpace.run { sqrt(norm()) }
```

Comment:
oops, accidentally deleted `.r`. Also, what should it mean?

Reply:
Euclidean distance, I think

Reply:
@altavir